src/ucp/wireup/wireup_cm.c

```diff
@@ -564,9 +564,12 @@ static unsigned ucp_ep_cm_disconnect_progress(void *arg)
     if (ucp_ep->flags & UCP_EP_FLAG_FAILED) {
         /* - ignore close event on failed ep, since all lanes are destroyed in
              generic err flow
-           - close request is valid only if all lanes are flushed, transport
-             error is unexpected */
-        ucs_assert(!(ucp_ep->flags & UCP_EP_FLAG_CLOSE_REQ_VALID));
+           - if close req is valid this is ucp_ep_close_nb request and it will
```

Comment:
"this is ucp_ep_close_nb request"
can you please mention why?

Reply:
because UCP_EP_FLAG_CLOSED is set, see assertion below, the comment is just description of statement below.

Reply:
so you mean because this flag is set only in ucp_ep_close_nbx. ok then.